pkg/controller/controller.go

```
@@ -250,7 +253,7 @@ func (p *csiProvisioner) Provision(options controller.VolumeOptions) (*v1.Persis
 			PersistentVolumeReclaimPolicy: options.PersistentVolumeReclaimPolicy,
 			AccessModes:                   options.PVC.Spec.AccessModes,
 			Capacity: v1.ResourceList{
-				v1.ResourceName(v1.ResourceStorage): options.PVC.Spec.Resources.Requests[v1.ResourceName(v1.ResourceStorage)],
+				v1.ResourceName(v1.ResourceStorage): resource.MustParse(repBytesString),
```

at least we need to validate the bytes are no less than what is claimed in the PVC.

---

@rootfs If number of bytes returned by the driver is less than in PVC's request, we fail, right?

---

I believe we should throw an error, I will add that in